platform/openide.util.lookup/src/org/openide/util/lookup/ProxyLookup.java

```diff
@@ -57,6 +58,26 @@
     public ProxyLookup(Lookup... lookups) {
         data = ImmutableInternalData.EMPTY.setLookupsNoFire(lookups, true);
     }
+    /**
+     * Create a ProxyLookup whose contents can be set dynamically without
+     * exposing the setter to consumers of the lookup; the passed
```

Comment:
The passed in `Consumer` is no longer called back. It can:

"be used later to change the set of lookups the `ProxyLookup` delegates to via `Controller.setLookups` method"